usaspending_api/references/management/commands/fix_zip_codes.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
+    help = 'One-time fixer to fill missing location.zip5 from zip4'
+
+    # Estimated runtime, based on simple linear extrap: 10 hours
+
+    logger = logging.getLogger('console')
+
+    update_qry = """
+       UPDATE references_location
+       SET    zip5 = SUBSTRING(zip4 FROM '^(\d{5})\-?(\d{4})?$')
+       WHERE  zip5 IS NULL
+       AND    zip4 IS NOT NULL
+    """
+
+    def handle(self, *args, **options):
```

Comment:
Adding log statements to show when the updates have started and finished, and any errors would be helpful.

Reply:
Agree with @shinson, this should print out every 10k rows as we do in our other broker scripts.

Reply:
Okay, latest commit breaks the update into 10k batches.